packages/tsx-editor/src/components/TsxEditor.tsx

```diff
@@ -138,15 +145,17 @@ function _loadTypes(supportedPackages: IPackageGroup[]): Promise<void> {
   const promises: Promise<void>[] = [];
   const typesPrefix = filePrefix + 'node_modules/@types';
 
-  // React must be loaded first
+  // React types must be loaded first (don't use import() to avoid potential bundling issues)
   promises.push(
-    // @ts-ignore: this import is handled by webpack
-    import('!raw-loader!@types/react/index.d.ts')
-      // raw-loader 0.x exports a single string, and later versions export a default.
-      // The package.json specifies 0.x, but handle either just in case.
-      .then((result: string | { default: string }) => {
+    new Promise<void>(resolve =>
+      require.ensure([], require => {
```

Comment:
Why don't we make sure that this is bundled as ESM instead of CommonJS so that this switch to require.ensure isn't needed?